README.md

```diff
@@ -59,23 +63,20 @@ repo contains the `pulumi` CLI, language SDKs, and core Pulumi engine, and indiv
 
 ## Welcome
 
-<img align="right" width="400" src="https://pulumi.io/images/quickstart/console.png" />
+<img align="right" width="400" src="https://www.pulumi.com/images/docs/quickstart/console.png" />
 
 * **[Getting Started](#getting-started)**: get up and running quickly.
 
-* **[Tutorials](https://pulumi.io/quickstart)**: walk through end-to-end workflows for creating containers, serverless
+* **[Tutorials](https://www.pulumi.com/docs/quickstart/)**: walk through end-to-end workflows for creating containers, serverless
```

Comment:
https://www.pulumi.com/docs/quickstart/ has the get started tutorials only now, whereas the rest of the tutorials are currently at https://www.pulumi.com/docs/reference/tutorials/ so I assume we want to link to the latter URL here.

We probably want to either delete the inline Getting Started section in this document and just link to https://www.pulumi.com/docs/quickstart/, or clean it up.

Reply:
Updated the link, though I'm inclined to keep the "Getting Started" section in the README.md file. Though cleaning it up sounds like a good thing to do.